docs/tutorials/from-tabular-data-to-binned-data.ipynb

```diff
@@ -420,42 +407,35 @@
     "This reflects neither the physics nor the logical structure of the detectors and is generally not very useful.\n",
     "The original table additionally contains information about the logical structure of the detector array.\n",
     "In this case it is divided into modules, segments, counters, wires, and strips.\n",
-    "We define:"
+    "We can use `group` to perform a binning based on discrete values.\n",
+    "The result is 5-D:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "fbc8d773-9271-46a6-8a39-eec7525f7463",
+   "id": "e800d4f1-5851-4394-af7c-bcf07d54e7d8",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Note that indices in the file are 1-based, not 0-based\n",
-    "groups = {\n",
-    "    dim: sc.arange(dim, 1, table.coords[dim].max().value + 1, unit=None, dtype='int64')\n",
-    "    for dim in ['module', 'segment', 'counter', 'wire', 'strip']\n",
-    "}\n",
-    "groups"
+    "binned_logical = table.group('module', 'segment', 'counter', 'wire', 'strip')\n",
+    "binned_logical"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "14a4ec01-4649-4dc3-ad2e-e92432e690e1",
-   "metadata": {},
-   "source": [
-    "Instead of using `sc.bin` with the `edges` keyword argument we can use the `groups` keyword argument to perform a binning based on discrete values.\n",
-    "The result is 5-D:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "e800d4f1-5851-4394-af7c-bcf07d54e7d8",
+   "id": "13686467-f6fa-4e68-9913-5f7a5e9af0b5",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "binned_logical = sc.bin(table, groups=list(groups.values()))\n",
-    "binned_logical"
+    "Above we used automatic group setup.\n",
+    "This will create a group for each unique value.\n",
+    "Explicit custom groups (given as a variable) should be considered in practice, for two reasons:\n",
```

Comment:
```suggestion
    "Explicit custom groups (given as a variable) should be considered carefully in practice, for two reasons:\n",
```

Or something like that? The sentence reads funny.

Also, I find this paragraph a little confusing. I don't think many people will understand what `Explicit custom groups (given as a variable)` will mean. Probably a code example is warranted here.
However, I think it's diverting the focus of the tutorial. I would suggest moving this paragraph to a footnote or something?

Reply:
@jl-wynen ?

Reply:
I agree that if we mention this here, there should be a code example and ideally an exercise. Could be something like getting only even wires.

I don't know how well this can fit into the tutorial. And I agree with Neil that is distracting. On the other hand, it should be explained somewhere. Would it be possible to have an optional section? I think a footnote makes it difficult to add code and exercise.

Reply:
We could use custom groups to "select" only some subset of the data?

Reply:
> Or something like that? The sentence reads funny.

Changed "considered carefully" -> favored

Reply:
Adding example extracting wires 1,2,3,4